swift-sdk/ui-components/uikit/IterableEmbeddedView.swift

```diff
@@ -32,8 +32,8 @@ public class IterableEmbeddedView:UIView {
     @IBOutlet weak var horizontalButtonStackViewSpacer: UIView!
     
     /// IterableEmbeddedView Image View.
-    @IBOutlet weak public var imgView: UIImageView!
-    @IBOutlet weak public var cardImageView: UIImageView!
+    @IBOutlet weak public var imgView: UIImageView?
```

Comment:
This is the fix right there! Good find! We should always expect image to not come just in case..